packages/template-typescript-minimal/package.json

```diff
@@ -36,11 +37,7 @@
     "npm": "^8.0.0 || ^9.0.0 || ^10.0.0"
   },
   "mobify": {
-    "app":{
-      "extensions": [
-        "test-extension"
-      ]
-    },
+    "app":{},
```

Comment:
I recently had to remove it because the @salesforce/extension-sample is currently private and when we generate a typescript minimal project it try's to install it from the local repo which it doesn't exist. 

If you refer to any of the spike doc's you'll see that it simply looks like 
```
app: {
    extensions: ['@salesforce/extension-sample']
}
```

Reply:
Is there no other way to test-drive this PR? How did you test the PR from your end?

I'm running the typescript-minimal project with the `extension-sample` extension, and I'm seeing this error on the page: 
`ModuleNotFoundError: Module not found: Error: Can't resolve '@salesforce/extension-sample/setup-app' in '/Users/vmarta/repos/pwa-kit/packages/pwa-kit-react-sdk/dist/ssr/universal/extensibility'`

Reply:
Sorry.. you have to have the npm package install in your package.json. 
I actually had it set up pretty much until 4 days ago before I removed it because of the aforementioned issue, I'll update the PR steps to call out what you have to do to test.

Reply:
I've update the PR, please see the steps to test.